lib/plugins/plugin/lib/utils.js

```diff
@@ -52,10 +52,13 @@ module.exports = {
   getPluginInfo(name) {
     let pluginInfo;
     if (name.startsWith('@')) {
-      pluginInfo = _.split(name.slice(1), '@', 2);
+      pluginInfo = name
+        .slice(1)
+        .split('@')
+        .slice(0, 2);
```

Comment:
We can do as in original version, and pass `2` as second argument to `split` (it's supported natively)

Reply:
Ah good catch @medikoo, I'll post an updated PR later today.